pkg/api/v1/types.go

```diff
@@ -519,25 +519,25 @@ type VolumeMount struct {
 type EnvVar struct {
 	// Required: This must be a C_IDENTIFIER.
 	Name string `json:"name" description:"name of the environment variable; must be a C_IDENTIFIER"`
-	// Optional: defaults to "".
+	// Optional: No more than one of the following may be set.
```

Comment:
These changes are because I copied v1beta3 to v1 again, so this now includes changes to v1beta3 that happened while https://github.com/GoogleCloudPlatform/kubernetes/pull/7454 was in review.